consts/validation.js

```diff
@@ -22,7 +22,8 @@ const enums = {
   COOPERATION_STATUS: ['pending', 'active', 'declined', 'closed'],
   PARAMS: ['id', 'categoryId', 'subjectId'],
   OFFER_STATUS: ['active', 'draft', 'closed'],
-  NEED_ACTION: ['tutor', 'student']
+  NEED_ACTION: ['tutor', 'student'],
+  NOTIFICATION_TYPE: ['chat', 'review', 'comment']
```

Comment:
if you removed chat from reference enum, remove it from this enum as well

Reply:
done.